src/Attribute/ConfirmationRoute.php

```diff
@@ -0,0 +1,38 @@
+<?php declare(strict_types=1);
+
+namespace App\Attribute;
+
+use Attribute;
+use Symfony\Component\Routing\Annotation\Route;
+
+#[Attribute(Attribute::TARGET_METHOD)]
+class ConfirmationRoute extends Route
+{
+    /**
+     * @param array|string      $data         data array managed by the Doctrine Annotations library or the path
+     * @param array|string|null $path
+     * @param string[]          $requirements
+     * @param string[]|string   $methods
+     * @param string[]|string   $schemes
+     */
+    public function __construct(
```

Comment:
you don't override anything from the constructor right? Then you can remove the constructor completely so the new class is empty and works just like the parent route

This has major benefits when there is a breaking change in the parent class, as we don't have to care about that here

same is also true for the RegistrationRoute

Reply:
Ah yes, you are  right :smile: